ocrolib/common.py

```diff
@@ -680,16 +680,23 @@ def ocropus_find_file(fname, gz=True):
     directories (as well as the current directory)."""
     possible_prefixes = [os.curdir]
 
-    local_share_path = os.path.dirname(inspect.getfile(inspect.currentframe()))
-    local_share_path = os.path.join(local_share_path,
-                                    os.pardir, os.pardir, os.pardir, os.pardir,
-                                    "share", "ocropus")
-    if os.path.isdir(local_share_path):
+    install_share_path = os.path.dirname(
+        inspect.getfile(inspect.currentframe()))
+    install_share_path = os.path.join(install_share_path, os.pardir, os.pardir,
+                                      os.pardir, os.pardir, "share", "ocropus")
+    if os.path.isdir(install_share_path):
+        possible_prefixes += [install_share_path]
+
+    local_share_path = os.path.join(sysconfig.get_config_var("prefix"),
```

Comment:
Won't this result in `/usr/local/local/share` if python is installed with `prefix=/usr/local`? Why not hard-wire `/usr/local/share/ocropus`? Or use `os.path.join(sysconfig.get_config_var('data'), 'share', 'ocropus')` though I'm not sure what this is set to if Python isn't installed with `prefix=/usr`?